From b5e8dc446572c1198a24eff105a394d0f307951e Mon Sep 17 00:00:00 2001
From: Solomon Peachy <pizza@shaftnet.org>
Date: Fri, 15 Apr 2016 23:05:43 -0400
Subject: [PATCH] build:  Don't allow USB backends to be built if CUPS is not
 enabled.

[Waldemar: backport from upstream commit
fc1713589fed645f2c8002c3f66dfb69af314716.]
Signed-off-by: Waldemar Brodkorb <wbx@openadk.org>
---
 configure.ac | 14 ++++++++------
 1 file changed, 8 insertions(+), 6 deletions(-)

diff --git a/configure.ac b/configure.ac
index 800794b..ba30019 100644
--- a/configure.ac
+++ b/configure.ac
@@ -442,12 +442,14 @@ dnl if test -z "${[EXTRA_LIBREADLINE_DEPS]}" ; then
 dnl   [EXTRA_LIBREADLINE_DEPS]=""
 dnl fi
 
-# libusb-1.0 (For CUPS backends)
-PKG_CHECK_MODULES([LIBUSB], [libusb-1.0],
-                  [HAVE_LIBUSB=yes
-                   BUILD_LIBUSB_BACKENDS=yes],
-                  [HAVE_LIBUSB=no
-	           BUILD_LIBUSB_BACKENDS=no])
+# libusb-1.0 (For CUPS backends, but only bother if CUPS is enabled)
+if test "x$BUILD_CUPS" = "xyes" ; then
+ PKG_CHECK_MODULES([LIBUSB], [libusb-1.0],
+                   [HAVE_LIBUSB=yes
+                    BUILD_LIBUSB_BACKENDS=yes],
+                   [HAVE_LIBUSB=no
+	            BUILD_LIBUSB_BACKENDS=no])
+fi
 
 AC_PATH_PROGS([GIMPTOOL2_CHECK], [gimptool-2.0 gimptool])
 if test -z "${GIMPTOOL2_CHECK}" ; then
-- 
2.6.4

